shared/chat/conversation/header-area/normal/index.native.js

```diff
@@ -61,10 +46,12 @@ const ShhIcon = props => (
 const ChannelHeader = (props: Props) => (
   <Wrapper {...props}>
     <Box2 direction="horizontal" style={styles.channelHeaderContainer}>
-      <Avatar teamname={props.teamName} size={16} />
+      <Avatar teamname={props.teamName} size={12} />
```

Comment:
should `size` be bigger for the small team header?  I see 16 in some designs
<img width="375" alt="image" src="https://user-images.githubusercontent.com/11968340/51545277-a1e5d100-1e2f-11e9-8994-b817c89ebe53.png">